mod-orders/mod-orders.postman_collection.json

```diff
@@ -0,0 +1,967 @@
   \"code\": \"GENRL\",\n          \"percentage\": 20.0,\n          \"encumbrance\": \"0466cb77-0344-43c6-85eb-0a64aa2934e5\",\n          \"po_line_id\": \"c0d08448-347b-418a-8c2f-5fb50248d67e\"\n        } \n      ],\n      \"location\": {\n        \"id\": \"{{$guid}}\",\n        \"location_id\": \"eb2d063a-5b4c-4cab-8db1-5fc5c5941df6\",\n        \"quantity\": 2,\n        \"quantity_electronic\": 1,\n        \"quantity_physical\": 1,\n        \"po_line_id\": \"8c778aee-97fa-4586-b131-3ea588a728e2\"\n      },\n      \"order_format\": \"Physical Resource\",\n      \"owner\": \"ABCDEFGHIJKLMNOPQRSTUVWXYZABC\",\n      \"payment_status\": \"Awaiting Payment\",\n      \"physical\": {\n        \"volumes\": 1,\n        \"id\": \"{{$guid}}\",\n        \"material_supplier\": \"73d14bc5-d131-48c6-b380-f8e62f63c8b6\",\n        \"receipt_due\": \"2018-10-10T00:00:00.000Z\",\n        \"po_line_id\": \"8c778aee-97fa-4586-b131-3ea588a728e2\"\n      },\n      \"po_line_description\": \"ABCDEFGHIJKLMNOPQRSTUVWXY\",\n      \"po_line_number\": \"268758-03\",\n      \"po_line_workflow_status\": \"Open\",\n      \"publication_date\": \"2017\",\n      \"publisher\": \"Schiffer Publishing\",\n      \"purchase_order_id\": \"d79b0bcc-DcAD-1E4E-Abb7-DbFcaD5BB3bb\",\n      \"receipt_date\": \"2018-10-09T00:00:00.000Z\",\n      \"receipt_status\": \"Awaiting Receipt\",\n      \"renewal\": {\n        \"interval\": 182,\n        \"id\": \"{{$guid}}\",\n        \"cycle\": \"6 Months\",\n        \"manual_renewal\": true,\n        \"review_period\": 30,\n        \"renewal_date\": \"2019-04-09T00:00:00.000Z\",\n        \"po_line_id\": \"8c778aee-97fa-4586-b131-3ea588a728e2\"\n      }, \n      \"reporting_codes\": [\n        {\n          \"code\": \"CODE1\",\n          \"id\": \"{{$guid}}\",\n          \"description\": \"ABCDEF\"\n        },\n        {\n          \"code\": \"CODE2\",\n          \"id\": \"{{$guid}}\",\n          \"description\": \"ABCDEFGHIJKLMNOPQRSTUVWXYZ\"\n        },\n        {\n          \"code\": \"CODE3\",\n          \"id\": \"{{$guid}}\",\n          \"description\": \"ABCDE\"\n        }\n      ],\n      \"requester\": \"Leo Bulero\",\n      \"rush\": true,\n      \"selector\": \"ABCD\",\n      \"source\": {\n        \"code\": \"ABCDEFGHIJKLMNOPQRSTUVWXYZABC\",\n        \"id\": \"{{$guid}}\",\n        \"description\": \"ABCDEFGHIJKLMNO\"\n      },\n      \"tags\": [\n        \"ABCDEFGHIJKLMNOPQRSTU\",\n        \"ABCDEFG\",\n        \"ABCDEFGHIJKLMNOPQRSTU\",\n        \"ABCDEFGHIJKLMNO\"\n      ],\n      \"title\": \"Kayak Fishing in the Northern Gulf Coast\",\n      \"vendor_detail\": {\n        \"id\": \"{{$guid}}\",\n        \"instructions\": \"ABCDEFG\",\n        \"note_from_vendor\": \"ABCDEFGHIKJKLMNOP\",\n        \"ref_number\": \"123456-78\",\n        \"ref_number_type\": \"Supplier's unique order line reference number\",\n        \"vendor_account\": \"8910-10\",\n        \"po_line_id\": \"8c778aee-97fa-4586-b131-3ea588a728e2\"\n      }\n    }\n  ]\n}"
```

Comment:
wouldn't it be better if we pulled the request from github rather than hardcoding, we can reuse the same request for any future API tests that we develop